### PR TITLE
Add normalized timezone name to RestWorkflowDefinition

### PR DESCRIPTION
**Pull request by frsyuki**

### CODE REVIEW

File: digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java [hunk 1]
```diff
@@ -19,6 +21,9 @@
 
     String getRevision();
 
+    @JsonProperty("timezone")
+    ZoneId getTimeZone();
```
Comment (komamitsu) [inline digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java]: The attribute name is `timezone`, so the getter should be `getTimezone()`?

**Comment by komamitsu** [comment]
I added one comment, but overall it looks good 👍

File: digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java (hunk 1 above)
Comment (frsyuki) [inline digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java]: I took the same code from https://github.com/treasure-data/digdag/blob/082f72b3b1adcbad570950544b60fc360835dfd3/digdag-client/src/main/java/io/digdag/client/api/RestWorkflowSessionTime.java#L19-L20

Comment (frsyuki) [inline digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java]: I think using `getTimeZone` is OK for consistency with others: https://github.com/treasure-data/digdag/search?utf8=%E2%9C%93&q=getTimeZone

**Comment by frsyuki** [comment]
👍

**Pull request merged**